### ISSUE TITLE
[Feature Request] Add prefix option to settings (`>> text` vs. `> > text`)

### ISSUE THREAD

**Issue by huyz** [czottmann/obsidian-blockquote-levels#1]
Would it be possible to add a feature to indent by adding `> ` with a space rather than just `>`?

**Comment by czottmann** [comment]
@huyz The first level invention always contains a space, subsequent ones do not. You ask for an  extra space on every level, then?

**Comment by huyz** [comment]
Yes exactly.

Like in these examples: https://github.com/chrisalley/markdown-garden/blob/master/source/guides/quotations/nested-block-quotes.md

```markdown
> This is a block quote containing another block
> quote which contains a header and a list.
>
> > # This is a header
> >
> > 1. Item One
> > 2. Item Two
```

Above, it makes a lot more sense to me that the `This` is aligned with the `> # this is a header`

**Comment by czottmann** [comment]
Sure thing, I'll add that.  :)  Give me a few days, please.  In the meantime, thanks for using it, and I'm happy about bug reports and feedback.

Quick unrelated question: are you using the plugin under macOS or Windows?

**Comment by huyz** [comment]
> Quick unrelated question: are you using the plugin under macOS or Windows?

macOS

**Comment by huyz** [comment]
@czottmann can you make a new release so I can try it out?

**Comment by czottmann** [comment]
I'll get to it in the next few days, and I'll keep you posted, @huyz.  Can't tell you more than that, sorry.  I've a few projects going at the moment and some are higher priority than this here plugin.

It **will** be done, that I can say. :)

**Comment by huyz** [comment]
@czottmann No problem, take your time. Was just pinging in case you forgot to make a release.

**Comment by czottmann** [comment]
Implemented in 2ec5390262fe9863bf199e6c356e6a1efc490042, rolled out in v1.1.0. 

The plugin still sits in the Community Plugins submissions queue (progress was made, tho!); the release is available in BRAT, naturally.
